pokerbot.js

```diff
@@ -131,6 +137,23 @@ pokerbot.root = function (req, res, next) {
       pokerbot.pokerDataModel[jiraId].channelId['id'] = channel.id
       pokerbot.pokerDataModel[jiraId].channelId['name'] = channel.name
       pokerbot.pokerDataModel[jiraId].channelId['members'] = channel.members.length
+      pokerbot.pokerDataModel[jiraId].channelId['membersList'] = []
+      pokerbot.pokerDataModel[jiraId].channelId['membersList'] = channel.members
+      let memberId
+      let miisingMemberArray = []
```

Comment:
Typo. should be `missingMemberArray` although there is no reason to use Hungarian notation. This variable should be `missingMembers`